### PR TITLE
improv(entity): Various fixes and tweaks for the Tempest

### PR DESCRIPTION
**Pull request by Zepalesque**
This pull request fixes a few issues and makes a few changes to the Tempests:
- The tempests now properly die during daytime (was broken since the pull request making them extend Zephyr)
- Tempests now hover up and down and have cloudlike particles underneath them, as they did in GoTV
- Tempests now have glowing tails, as they were _intended_ to in GoTV

Here is a video showing the new tweaks to their appearance and animation:

https://github.com/The-Aether-Team/The-Aether-Genesis/assets/60141811/3168eba1-293c-4219-ae67-f6c2bdf46609

---

I agree to the Contributor License Agreement (CLA).

### CODE REVIEW

File: src/main/java/com/aetherteam/aether_genesis/event/listeners/EntityListener.java [hunk 1]
```diff
@@ -11,15 +13,17 @@
 public class EntityListener {
     @SubscribeEvent
     public static void DisabledNightSpawn(MobSpawnEvent.FinalizeSpawn event){
-        LivingEntity zephyr = event.getEntity();
-        if(zephyr instanceof Zephyr && !zephyr.getLevel().isClientSide && zephyr.getLevel().isNight())
+        LivingEntity entity = event.getEntity();
+        if(entity.getType() == AetherEntityTypes.ZEPHYR.get() && !entity.getLevel().isClientSide && entity.getLevel().isNight())
+            event.setSpawnCancelled(true);
+        if(entity.getType() == GenesisEntityTypes.TEMPEST.get() && !entity.getLevel().isClientSide && !entity.getLevel().isNight())
```
Comment (bconlon1) [inline src/main/java/com/aetherteam/aether_genesis/event/listeners/EntityListener.java]: The tempest spawn handling should already be handled in the tempest class.

Comment (Zepalesque) [reply src/main/java/com/aetherteam/aether_genesis/event/listeners/EntityListener.java]: ah, ok

File: src/main/java/com/aetherteam/aether_genesis/client/renderer/entity/TempestTransparencyGlowLayer.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        super(entityRenderer);
+        this.transparency = transparencyModel;
+    }
+
+    @Override
+    public RenderType renderType() {
+        return LAYER_GLOW;
+    }
+
+    /*    @Override
```
Comment (bconlon1) [inline src/main/java/com/aetherteam/aether_genesis/client/renderer/entity/TempestTransparencyGlowLayer.java]: does this code need to be here if its commented out?

Comment (Zepalesque) [reply src/main/java/com/aetherteam/aether_genesis/client/renderer/entity/TempestTransparencyGlowLayer.java]: ah oops, I thought I removed that

**Comment by Zepalesque** [comment]
hmm, I just remembered something _someone_ said, I can't remember who, but the spawn egg colors are wrong, is that something that would be a good idea to change, or should I keep it as it is now?

**Review by bconlon1 — APPROVED** [review]
lgtm

**Pull request merged**